File: samples/java/example13/NumbersTester.java
//HIDDEN
public class NumbersTester
{
   public static void main(String[] args)
   {
      Numbers nums = new Numbers();
      System.out.println(nums.square(3));
      System.out.println("Expected: 9");
      System.out.println(nums.square(-3));
      System.out.println("Expected: 9");
   }
}

File: samples/rust/example2/numbers_tester.rs
mod numbers;

fn main() {
    println!("{:?}", numbers::square(3));
    println!("Expected: 9");
    println!("{:?}", numbers::square(-3));
    println!("Expected: 9");
    println!("{:?}", numbers::square(0));
    println!("Expected: 0");
}